config/default.yml

```diff
@@ -711,6 +711,17 @@ PreCommit:
     install_command: 'npm install -g standard'
     include: '**/*.js'
 
+  StyleLint:
+    enabled: true
+    description: 'Check styles with Stylelint'
+    require_executable: 'stylelint'
+    flags: ['-f compact']
```

Comment:
These need to be split into separate arguments `['-f', 'compact']`, unless the `stylelint` CLI accepts `-f compact` as a single argument, but that would be surprising.